provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -1306,6 +1365,17 @@ rules:
   - update
   - delete
   - patch
+- apiGroups:
+  - "aci.droplog"
+  resources:
+  - droplogs
+  verbs:
+  - list
+  - watch
+  - get
+  - create
```

Comment:
Does the hostagent really need create/update/delete permissions for droplogs? I know our other CRDs are configured this way now but list/watch/get seems like all that's needed

Reply:
Yes, list/watch/get should be enough. Will update that, thanks!